modules/planning/scenarios/bare_intersection/unprotected/stage_approach.cc

```diff
@@ -41,6 +41,11 @@ Stage::StageStatus BareIntersectionUnprotectedStageApproach::Process(
 
   scenario_config_.CopyFrom(GetContext()->scenario_config);
 
+  if (!config_.enabled()) {
+    ADEBUG << "stage Approach finished because not enable.";
+    return Stage::FINISHED;
+  }
+
```

Comment:
this is not how it works.
for each stage, after it finishes, it shall set next_stage_ , for the stage machine to move on. 
only setting up stage status doesn't help in that way. 
And secondly, although we have enable flag as conf for stages, we'd better not allow skip stages in general in code implementation, to avoid bad conf by mistakes.

Reply:
Thanks for your reminding.